src/app/inventory/item-types.ts

```diff
@@ -151,6 +151,8 @@ export interface DimItem {
   tracked: boolean;
   /** Is this item locked? */
   locked: boolean;
+  /** Is this item favorited? */
+  favorited: boolean;
```

Comment:
Let's not add a new property for this - instead, we can switch the icon at display time. That will make this a much smaller change.

Reply:
Sure thing, works for me!